notebooks/tutorials/data-scientist/06-messaging-and-requests.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>you can just do pointer.request(client) here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>